xtuner/v1/module/attention/mha.py

```diff
@@ -431,3 +438,11 @@ def build_kv_cache(
         cache_v = torch.zeros(num_blocks, block_size, num_heads, head_dim, dtype=dtype, device="cuda")
 
         return cache_k, cache_v
+
+    @property
+    def name(self):
```

Comment:
https://google.github.io/styleguide/pyguide.html#2134-decision

I'd like to suggest following this